scipy/optimize/_dual_annealing.py

```diff
@@ -228,9 +228,9 @@ class StrategyChain(object):
         Instance of `ObjectiveFunWrapper` class.
     minimizer_wrapper: LocalSearchWrapper
         Instance of `LocalSearchWrapper` class.
-    rand_state : RandomState object
-        A numpy.random.RandomState object for using the current state of the
-        created random generator container.
+    rand_state : `~numpy.random.mtrand.RandomState` object
+        A `~numpy.random.mtrand.RandomState` object for using the current state
```

Comment:
What is ~ for?

Reply:
Trims everything except what follows the last period in the ref (look at the rendered doc and you should see it)